### ISSUE TITLE
Icon Sizing Problem

### ISSUE THREAD

**Issue by MrSkemo** [duarteocarmo/coronabar#10]
When running the program the icon is very large and omits everything else in my taskbar.
I'm using Ubuntu 18.04 and installed the app via the Snap Store

**Comment by duarteocarmo** [comment]
@lapisdecor any idea? 

@MrSkemo do you mind providing more details? Are you using GNOME, etc..? Perhaps a screenshot would be helpful as well!

**Comment by MrSkemo** [comment]
I believe the main system is based on Budgie, but there is some GNOME in it too. Unfortunately I don't quite know how it is set up exactly since it's some custom build.

This is how the the bar looks when the app is open. It's technically still functional but since the crornabar logo is supersized all the other icons disappear
![Coronabar Taskbar](https://user-images.githubusercontent.com/37907402/97919993-064df800-1d59-11eb-90ac-7a5acc648670.png)

Edit. some spelling errors

**Comment by duarteocarmo** [comment]
Hey @MrSkemo, the snap version was based on a pure GNOME shell with plain Ubuntu. 

Would love to make this package available to you - but unfortunately it's out of the scope of the project to port this application to all distributions/desktop environments/tray systems in the Linux ecosystem.. This is why we included [this](https://github.com/duarteocarmo/coronabar/blob/master/README.md#linux) note. 

Closing this for now.. But thanks for the report!
